CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# HERMES development workflow
+
+## Preamble
+
+> **Branching is cheap!**
+>
+> Work in small packets, put up small pull requests!
+>
+> Aim for quick turnaround times!
+
+## Branching
+
+We loosely follow a mixture of [GitHubFlow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/):
+
+`main`
+    - Stable branch
+    - Merges come only from `develop` or a hotfix branch (i.e., when something needs to be fixed in "production")
+
+`develop`
+    - Unstable development branch
+    - Last "stable" is tagged: tag before attempting to break something
+
+`feature/<describe-feature>` (including issue id when exists: `feature/62-fix-broken-thing`)
+    - Branch from last tag on `develop`
+    - Feature branches may pick up "older" branches if they are not to complicated
+    - Combined branches must all start from the same tag
+
+### Pull requests
+
+- As soon as you have made 1 commit in a feature branch, put up a *draft* pull request
+- Keep pull requests small :skull: 
+- :warning: No pre-emptive reviews on PR drafts, **unless** the PR author @-mentions with this specific request
+- When you think you're done, mark PR ready for review, see below (Merge Process)
+
+## Merge Process (into `develop`)
+
+- Create PR from `feature/...` against `develop` (PR author)
+- Describe work in initial comment (PR author)
```

Comment:
**suggestion**: it might be helpful to create a PR template and include in this PR.

Reply:
All for it. Feel free to put up a draft PR 🙂.

Reply:
I'd simply push into this PR to get it into the discussion instead of opening a new place to look at, agreed @sdruskat ?

Reply:
TBH, I'd rather have this separate to avoid this being open longer than necessary. If you don't mind.

Your PR would probably also make changes to this doc, so it'd be nice to have it in place first.

Reply:
Yes, unmerged branches are a burden and lead to more complex new branches... ;)

Reply:
Created #47 for this, resolving here.